### PR TITLE
Minor sessions/app password improvements

### PR DESCRIPTION
**Pull request by icewind1991**
- Improved handling of safari user agents
- Changed wording from "Recent activity" to "Last activity"
- Improved styling of tables a bit

Before

After

cc @nextcloud/designers

### CODE REVIEW

**Comment by LukasReschke** [comment]
👍

**Comment by icewind1991** [comment]
Made sure tokens are always sorted by time

File: settings/js/authtoken_view.js [hunk 1]
```diff
@@ -77,6 +77,10 @@
 			// Show header only if there are tokens to show
 			this._toggleHeader(tokens.length > 0);
 
+			tokens = tokens.sort(function(a, b) {
+				return b.get('lastActivity') - a.get('lastActivity');
+			});
+			console.log(tokens);
```
Comment (ChristophWurst) [inline settings/js/authtoken_view.js]: remove this line please :-)

File: settings/js/authtoken_view.js [hunk 2]
```diff
@@ -77,6 +77,10 @@
 			// Show header only if there are tokens to show
 			this._toggleHeader(tokens.length > 0);
 
+			tokens = tokens.sort(function(a, b) {
```
Comment (ChristophWurst) [inline settings/js/authtoken_view.js]: alternatively, ``sortBy: 'lastActivity'`` on the collection would also sort the items according to http://backbonejs.org/#Collection-comparator

Comment (icewind1991) [inline settings/js/authtoken_view.js]: nice, thanks for the tip

**Comment by MorrisJobke** [comment]
👍

**Pull request merged**